sources/AppBundle/Event/Talk/InvitationFormHandler.php

```diff
@@ -65,7 +65,7 @@ public function handle(Request $request, Event $event, FormInterface $form, Gith
         // Send mail to the other guy, begging for him to join the talk
         $this->eventDispatcher->addListener(KernelEvents::TERMINATE, function () use ($talk, $user, $event, $invitation) {
             $text = $this->translator->trans('mail.invitation.text', [
-                '%login%' => $user->getLogin(),
+                '%name%' => strlen($user->getName()) > 0 ? $user->getName() : $user->getLogin(),
```

Comment:
```suggestion
                '%name%' =>  $user->getName() ?: $user->getLogin(),
```
`mb_strlen` devrait être utilisé plutôt que `strlen`, mais dans ce cas précis on n'a pas vraiment besoin de connaître la longueur du nom.

Reply:
Done